JsonLd/Response.php

```diff
@@ -18,7 +18,7 @@
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
-class JsonLdResponse extends JsonResponse
+class Response extends JsonResponse
```

Comment:
Importing `Response` should not be necessary for using constants and so on because `JsonLdResponse` extends it.
You think when you mix controller returning HTML and JSON-LD?